### PR TITLE
new methods for temporal raster aggregator

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Rasteraggregation: Count + Summe](https://trello.com/c/N6BMAeN4/821-rasteraggregation-count-summe)

**Review by jdroenner — CHANGES_REQUESTED** [review]

File: operators/src/processing/temporal_raster_aggregation/subquery.rs [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            }
+        };
+
+        match &mut self.state_grid {
+            GridOrEmpty::Empty(_) if !self.prestine && !F::IGNORE_NO_DATA => {
+                // every pixel is nodata we will keep it like this forever
+            }
+
+            GridOrEmpty::Empty(_) => {
+                // TODO: handle case where this could stays empty
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply operators/src/processing/temporal_raster_aggregation/subquery.rs]: [ceee342](https://github.com/geo-engine/geoengine/pull/689/commits/ceee3420733edd171b699a198456b1f21674d8a7)

Actually, I don't know. This is just the current implementation, I just copied it.

**Comment by jdroenner** [comment]
bors merge

**Comment by jdroenner** [comment]
bors retry

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**